webdriver/tests/support/pdf.py

```diff
@@ -4,5 +4,6 @@
 def assert_pdf(value):
     data = decodebytes(value.encode())
 
-    assert data.startswith(b"%PDF-"), "Decoded data starts with the PDF signature"
-    assert data.endswith(b"%%EOF\n"), "Decoded data ends with the EOF flag"
+    assert data.startswith(b"%PDF-"), "Decoded data should start with the PDF signature"
+    # TODO: removed `data.endswith(b"%%EOF")` after https://crbug.com/1430371 is fixed.
+    assert data.endswith(b"%%EOF\n") or data.endswith(b"%%EOF"), "Decoded data should end with the EOF flag"
```

Comment:
```suggestion
    # TODO: Remove `data.endswith(b"%%EOF")` after https://crbug.com/1430371 is fixed.
    assert data.endswith(b"%%EOF\n") or data.endswith(b"%%EOF"), "Decoded data should end with the EOF flag"
```

Actually we could also use `rstrip` to get rid of the final character if it's a carriage return, so that we do not exactly test how a line is defined in the "PDF spec", but what's actually the content of that line. What do you think?

Reply:
It's great the check was that strict and allowed us to identify the issue on our side. So I'd prefer to have only this exception with a clear TODO, so that other implementations don't miss that detail.